### DISCUSSION TITLE
Are there additional Plotting parameters for Allmaps?

### DISCUSSION THREAD

**Discussion by jguppy** [tanghaibao/jcvi#443] [Q&A]
Hi, 

I was wondering if there was a way to add labels the SNP names, and also add cM tick marks in Allmaps plots at all?

I've had a look through the allmaps code and could find any flags to add these in.

Thanks for your help,

**Comment by tanghaibao** [comment]
@jguppy 

Currently no, although it won't be too difficult to add such support. 
One issue though, for most maps, is that the labels would be too dense to be legible.
